css/style.css

```
@@ -0,0 +1,565 @@
+/* Custom properties */
+:root {
+  --basic-red: #ee3643;
+  --basic-green: #63a63e;
+  --basic-grey: #3d546f;
+  --basic-greylight: #c5c5c5;
+  --alt-grey: #32425c;
+  --alt-grey-light: #a9a9a9;
+  --basic-background: #f1f5f7;
+  --alt-background: #293449;
+  --title-background: #f9f5f0;
+  --white: #ffffff;
+  --black: #000000;
+  --special-yellow: #ffbf47;
+  --special-blue: #3bbce3;
+  --special-lilac: #dc91d8;
+  --special-greenlight: #8ed78f;
+  --special-yellowlight: #ffd180;
+  --success: #5fbb2d;
+  --warning: #ffbf47;
+  --error: #ba2732;
+  --hover-button-red: #ca2c37;
+  --active-link-white: rgba(255, 255, 255, 0.5);
+  --active-special-yellowlight: rgba(255, 209, 128, 0.5);
+  --active-nav-grey: #1d2739;
+  --active-buy-button-green: #518534;
+  --grey-lighten: #ffffff @0% rgba(255, 255, 255, 0) @100%;
+}
+
+/* FONTS */
+
+/* Cuprum-Regular */
+@font-face {
+  font-family: "Cuprum";
+  font-style: normal;
+  font-weight: 400;
+  src: local("Cuprum Regular"), local("Cuprum-Regular"),
+    url("../fonts/cuprum.woff2") format("woff2"),
+    url("../fonts/cuprum.woff") format("woff");
+}
+
+/* Cuprum-Bold */
+@font-face {
+  font-family: "Cuprum";
+  font-style: normal;
+  font-weight: 700;
+  src: local("Cuprum Bold"), local("Cuprum-Bold"),
+    url("../fonts/cuprumbold.woff2") format("woff2"),
+    url("../fonts/cuprumbold.woff") format("woff");
+}
+
+/* Cuprum-Italic */
+@font-face {
+  font-family: "Cuprum-Italic";
```

не исправлено, нужно так:
```suggestion
  font-family: "Cuprum";
```